tests/test_all.py

```diff
@@ -148,6 +149,9 @@ def test_lifecycle(auto_container):
                 support_end = datetime.datetime.strptime(
                     entry_date, "%Y-%m-%d"
                 )
+                if entry_name.startswith("go1.19-openssl"):
+                    # bsc#1216578
+                    continue
```

Comment:
How about something like this? Or is there more lifecycle data to be tested?
```suggestion
                if entry_name.startswith("go1.19-openssl"):
                    pytest.skip(reason="bsc#1216578")
```

Reply:
@dcermak that would abort the loop, we still want the other entries to be tested. I'd have to create a dict with fails and then diff that with the known exceptions. 

Not sure it is worth the effort, but I can do it. I don't plan on adding additional exceptions, I'd rather push people to provide correct data (and get QA to test stuff before releasing it)

Reply:
then nevermind